src/includes/performance-monitoring/configuration/vue.mdx

```diff
@@ -1,49 +1,54 @@
-When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
-and create a new activity named `Vue Application Render`. Once the activity has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the activity as completed.
+When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM BrowserTracing integration,
+and create a new span named `Vue Application Render`. Once the span has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the span as finished.
 
 The described instrumentation functionality will give you very high-level information about the rendering performance of the Vue instance. However, the integration can also provide more fine-grained details about what actually happened during a specific activity.
 To do that, you need to specify which components to track and what hooks to listen to (you can find a list of all available hooks [here](https://vuejs.org/v2/api/#Options-Lifecycle-Hooks)). You can also turn on tracking for all the components. However, it may be rather noisy if your app consists of hundreds of components. We encourage being more specific. If you don't provide hooks, Sentry will track a component's `mount` and `update` hooks.
 
 Note that we don't use `before` and `-ed` pairs for hooks, and you should provide a simple verb instead. For example, `update` is correct. `beforeUpdate` and `updated` are incorrect.
 
-To set up the Vue Tracing Integration, you will first need to configure the AM Tracing integration itself. For details on how to do this, see the [JavaScript](/performance-monitoring/distributed-tracing/#javascript) section above.
-Once you've configured the Tracing integration, move on to configuring the Vue integration itself.
+To set up the Vue Tracing Integration, you will first need to configure the AM BrowserTracing integration itself. For details on how to do this, see the [JavaScript](/performance-monitoring/distributed-tracing/#javascript) section above.
+Once you've configured the BrowserTracing integration, move on to configuring the Vue integration itself.
 Sentry built the new tracing capabilities into the original Vue error handler integrations, so there is no need to add any new packages. You only need to provide an appropriate configuration.
 
 The most basic configuration for tracing your Vue app, which would track only the top-level component, looks like this:
 
 ```js
 import * as Sentry from "@sentry/browser";
 import { Vue as VueIntegration } from "@sentry/integrations";
-import { Integrations } from "@sentry/apm";
+import { Integrations } from "@sentry/tracing";
```

Comment:
If we're importing the entire tracing package here so that the `Sentry.init()` can look the same as it does everywhere else, perhaps we should do the same for the Vue integration? It might not be a bad idea to have at least one example where we're gracefully handling the `Integrations` naming conflict.

(Alternatively, we could handle every instance of this the other way, and only ever import the `BrowserTracing` integration instead of the whole package, which would make the question of naming conflicts moot.)

Reply:
> only ever import the BrowserTracing integration instead of the whole package

We can make this change.